app/models/latest_news.rb

```diff
@@ -0,0 +1,3 @@
+class LatestNews < ApplicationRecord
```

Comment:
Add/delete/update all work great! One comment I have is maybe we can make the 'title' and 'priority' fields required, to help admins avoid adding empty/ news items without an order or keep track of the order they should be displayed in with these two lines:

```
validates :title, presence: true
validates :priority, presence: true
```

Reply:
yep good idea!